### PR TITLE
Use the pytorch-grad-cam tool to visualize Class Activation Maps (CAM).

### PR DESCRIPTION
**Pull request by zhen6618**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Use the pytorch-grad-cam tool to visualize Class Activation Maps (CAM).

## Modification

Use the pytorch-grad-cam tool to visualize Class Activation Maps (CAM), add a file named visualization_cam.py to tools/analysis_tools/ folder, add a test image car.jpg, a prediction image prediction.png, a cam image vis_cam.png to the main folder. 

Before running, download a weight file:  deeplabv3_r50-d8_769x769_40k_cityscapes.pth at https://github.com/open-mmlab/mmsegmentation/tree/main/configs/deeplabv3.

Run command: python tools/analysis_tools/visualization_cam.py

Then, you will get a prediction image prediction.png and a cam image vis_cam.png.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Comment by xiexinch** [comment]
Hi @zhen6618,
Thanks for your contribution! Please sign the CLA and fix the lint error, we'll test your PR and merge it ASAP.

**Comment by zhen6618** [comment]
> Hi @zhen6618, Thanks for your contribution! Please sign the CLA and fix the lint error, we'll test your PR and merge it ASAP.

@xiexinch There are no lint errors. But there is an error about mininum_version_gpu, i do not know how to handle it.

File: car.jpg
Comment (xiexinch) [inline car.jpg:1]: Might remote this image.

File: prediction.png
Comment (xiexinch) [inline prediction.png:1]: Might remote this image.

File: vis_cam.png
Comment (xiexinch) [inline vis_cam.png:1]: Might remote this image.

**Comment by xiexinch** [comment]
Hi @zhen6618,
We're looking for what happened, a new PR should do pr_stage_test instead of merge_stage_test.

**Comment by zhen6618** [comment]
Hi@xiexinch, 
Thanks a lot, any new discoveries to solve this problem? This is the first time I submit a PR, I tried to solve this problem, but I couldn't find a good way in the end.

**Comment by OpenMMLab-Assistant-004** [comment]
Hi @zhen6618,

We'd like to express our appreciation for your valuable contributions to the mmsegmentation. Your efforts have significantly aided in enhancing the project's quality.
It is our pleasure to invite you to join our community thorugh Discord_Special Interest Group (SIG) channel. This is a great place to share your experiences, discuss ideas, and connect with other like-minded people. To become a part of the SIG channel, send a message to the moderator, OpenMMLab, briefly introduce yourself and mention your open-source contributions in the #introductions channel. Our team will gladly facilitate your entry. We eagerly await your presence. Please follow this link to join us: ​https://discord.gg/UjgXkPWNqA.

If you're on WeChat, we'd also love for you to join our community there. Just add our assistant using the WeChat ID: openmmlabwx. When sending the friend request, remember to include the remark "mmsig + Github ID".

Thanks again for your awesome contribution, and we're excited to have you as part of our community!

**Comment by xiexinch** [comment]
Hi @zhen6618,
Could you follow our [CONTRIBUTION GUIDE](https://github.com/open-mmlab/mmsegmentation/blob/main/.github/CONTRIBUTING.md#workflow) to create a new PR? I have no idea how to solve it.

**Comment by zhen6618** [comment]
@xiexinch 
Okay. Thanks.

**Pull request closed without merging**